app/src/main/java/com/example/xpvehicles/activities/UserOwnedVehicleRequestsActivity.java

```diff
@@ -106,4 +97,28 @@ public void done(List<RentVehicle> requests, ParseException e) {
             }
         });
     }
+
+    private void setValues() {
+        final String DAILY_PRICE_PREFIX = "Daily Price: $";
```

Comment:
constants should be like following:

  public final class File {
    public static final int MIN_ROWS = 1;
    public static final int MAX_ROWS = 1000;

    private File() {}
  }